packages/design-system/src/styles/components/_Button.scss

```diff
@@ -135,27 +139,28 @@
     border-color: rgba($border-color-inverse, 0.8);
     color: rgba($color-base-inverse, 0.8);
   }
-
-  &:focus,
-  &.ds-c-button--focus {
-    background-color: $focus-color;
-    border-color: transparent;
-    box-shadow: $focus-shadow-link-inverse;
-    color: $color-base;
-    opacity: 1;
+  @if $ds-include-focus-styles {
+    &:focus,
+    &.ds-c-button--focus {
+      background-color: $focus-color;
+      border-color: transparent;
+      box-shadow: $focus-shadow-link-inverse;
+      color: $color-base;
+      opacity: 1;
+    }
   }
```

Comment:
We need an else statement here for [this](https://github.com/CMSgov/design-system/pull/510/files#diff-08998df4528f2cb9cfdc571c93ad3ce6L100)

Reply:
It looked to me like this block of code should remain regardless of if the focus styles are enabled or not.
```
  &:active,
  &.ds-c-button--active {
    border-color: rgba($border-color-inverse, 0.6);
    color: rgba($color-base-inverse, 0.6);
  }
```

Reply:
```
else {
   &:focus,
   &.ds-c-button--focus {
       border-color: rgba($border-color-inverse, 0.8);
       color: rgba($color-base-inverse, 0.8);
   }
}
```